utils/test/cypress.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Same re other config - I'd prefer it at top level if possible

Reply:
So I've had chats with @matthew-chirgwin before and I thought we wanted to avoid having too many top-level configs because it gets messy?

Reply:
We've got 9 linting configs that have been tucked away into a folder (with some being symlinked)

Reply:
@nictownsend I set the precident of having these configs in `utils/<type>` with the linting and build config. It helps wehere configs extend/build upon other items. I would prefer to keep it this way, and keep the top level/root of the repo as setup/dependency type items